pkgs/servers/sql/postgresql/ext/pipelinedb.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, postgresql, zeromq, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "pipelinedb";
+  version = "1.0.0-13";
+
+  src = fetchFromGitHub {
+    owner = "pipelinedb";
+    repo = pname;
+    rev = version;
+    sha256 = "1mnqpvx6g1r2n4kjrrx01vbdx7kvndfsbmm7zbzizjnjlyixz75f";
+  };
+
+  buildInputs = [ postgresql openssl zeromq ];
+
+  makeFlags = [ "USE_PGXS=1" ];
+
+  preConfigure = ''
+    substituteInPlace Makefile \
+      --replace "/usr/lib/libzmq.a" "${zeromq}/lib/libzmq.a"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
It seems like you install nothing in $out/bin

Reply:
This is required because an issue with PostgreSQL extraPlugins https://github.com/NixOS/nixpkgs/issues/22653; fwiw I have another PR to address it https://github.com/NixOS/nixpkgs/pull/58251